spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "my.custom.url": "vote.the-order.xyz"
```

You will need to add your custom URL
```suggestion
  "": "vote.the-order.xyz"
```